packages/owl-bot-post-processor/src/index.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+export * from './helpers';
```

Comment:
I don't understand with index.ts exists.

Reply:
I like having an `index.ts` in case you're playing with the library from the command line:

```
Welcome to Node.js v12.18.4.
Type ".help" for more information.
> require('./')
{
  triggerBuild: [Getter],
  getHeadCommit: [Getter],
  createCheck: [Getter],
  getToken: [Getter],
  getAccessTokenURL: [Getter],
  getAuthenticatedOctokit: [Getter],
  helpers: [Getter]
}
```

☝️ if I keep exporting method in the index like this, there's easy access from the Node repl.